API/controllers/ordersControllers.js

```diff
@@ -0,0 +1,275 @@
+const {
+  orderSchema,
+  orderEditSchema,
+  orderItemsSchema,
+} = require("../validators/validators");
+
+//GET ALL ORDERS
+function getOrders(req, res) {
+  let pool = req.pool;
+  pool.query(`select * from orders`, (err, result) => {
+    if (err) {
+      res.status(500).json({
+        success: false,
+        message: "Internal server error.",
+      });
+      console.log("Error occured in query", err);
+    }
+    if (result.rowsAffected[0] === 0) {
```

Comment:
zero defaults to false

Plus when selecting data I don't believe this rowsAffected will have a value greater than zero